### PR TITLE
xfwm4-themes: init at 4.10.0

### PR DESCRIPTION
**Pull request by volth**
###### Motivation for this change

Additional themes for XFCE.

The package is 5 years old and has version of 4.10 (actual XFCE is 4.12), yet there is nothing to update, it is basically graphic files of classic themes like Motif, Windows95, WindowsXP, etc
Ubuntu and other distros has the [same](http://packages.ubuntu.com/search?keywords=xfwm4-themes) version 

It is not included in default XFCE of NixOS and won't bloat any existing installation.
```pkgs.xfce.xfwm4themes``` should be explicitly added into ```environment.systemPackages``` for  extra themes to appear in the ```Window Manager``` settings box:

xfwm4-themes

###### Things done

- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by matthiasbeyer** [comment]
Hi. I like this change, although we might want to be able to configure the package to only contain _some_ of the themes - what do you think?

**Comment by volth** [comment]
@matthiasbeyer: there is a nix feature for radical customisation of any package:
```
      yourpackage = pkgs.existingpackage.overrideDerivation (old: {
        postInstall = old.postInstall + ''
          rm -rf $out/files-you-do-not-like
        '';
      });
```
it could be used to add or alter files as well.

File: pkgs/desktops/xfce/art/xfwm4-themes.nix [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  ver_maj = "4.10";
+  ver_min = "0";
+
+  src = fetchurl {
+    url = "mirror://xfce/src/art/${p_name}/${ver_maj}/${name}.tar.bz2";
+    sha256 = "0xfmdykav4rf6gdxbd6fhmrfrvbdc1yjihz7r7lba0wp1vqda51j";
+  };
+  name = "${p_name}-${ver_maj}.${ver_min}";
+
+  meta = {
```
Comment (rasendubi) [inline pkgs/desktops/xfce/art/xfwm4-themes.nix]: What's license?

Comment (volth) [inline pkgs/desktops/xfce/art/xfwm4-themes.nix]: I copied ```meta``` from the next file https://github.com/NixOS/nixpkgs/blob/master/pkgs/desktops/xfce/art/xfce4-icon-theme.nix
Reviewing and fixing xfce livenses may be the topic of another PR, as there are many .nix files in xfce dir lack license. And even those which have license in meta are not consistent (there are gpl2, gpl2plus and gpl3, although they are part of the same project)

Comment (rasendubi) [inline pkgs/desktops/xfce/art/xfwm4-themes.nix]: The source archive has COPYING, which is GPLv3.

Comment (matthiasbeyer) [inline pkgs/desktops/xfce/art/xfwm4-themes.nix]: Just because there are many files which do not have a license, you don't need to introduce even more files without a license! License information has to be part of every new package PR! Thanks for adding it!

Comment (volth) [inline pkgs/desktops/xfce/art/xfwm4-themes.nix]: I meant if you feel it is important, one could write a script which downloads ```src``` of every derivation in nixpkgs and check ```meta.license``` against the strings in the text files in the source tarball and then make PRs fixing licenses.
I'd expect there are too many files to fix so it should be a bot job.
